Common/Orders/Order.cs

```diff
@@ -285,5 +285,9 @@ public static Order CreateOrder(SubmitOrderRequest request)
             }
             return order;
         }
+        /// <summary>
+        /// Order Expiry on a specific UTC time.
+        /// </summary>
+        public DateTime DurationValue;
```

Comment:
@jaredbroad -- thoughts on this?  I think implementing it wouldn't be too bad, each time we do a scan in `BacktestingBrokerage.Scan` we would just need to check that we haven't passed this time, this is a fairly general way of expressing expiry, maybe change the name, turn it into a property with internal set like others.